recipes/fisher/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "fisher" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7519d2c884be6f5844bfde900fd294e98a33fe9a943c6b03800513cd281859c3
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - numpy
+    - cython
+    - pytest
+    - pip
+  run:
+    - python
+    - numpy
+    - cython
+    - pytest
+    - pip
+
+test:
+  imports:
+    - fisher
+
+about:
+  home: http://github.com/brentp/fishers_exact_test
+  license: BSD
+  license_family: BSD
+  license_file: ''
```

Comment:
I've raised an issue upstream related to the `license_file`:

https://github.com/brentp/fishers_exact_test/issues/21

Please put the above link as a comment in the `meta.yaml` as a reminder that this issue is pending and may be resolved in a future release.